app/javascript/components/subject/MoveToHousehold.js

```diff
@@ -73,52 +235,84 @@ class MoveToHousehold extends React.Component {
+                <Form.Label>
+                  Please select the new monitoree that will respond for <b>{this.getPatientName()}</b>.
                 </Form.Label>
-                <Form.Control as="select" className="form-control-lg" id="hoh_selection" onChange={this.handleChange} defaultValue={-1}>
-                  <option value={-1} disabled>
-                    --
-                  </option>
-                  {this.state?.dependents?.map((member, index) => {
-                    return (
-                      <option key={`option-${index}`} value={member.id}>
-                        {member.last_name}, {member.first_name} Age: {member.age}, State ID: {member.state_id}
-                      </option>
-                    );
-                  })}
-                </Form.Control>
+                <p>
+                  You may select from the provided existing Head of Households and monitorees who are self reporting.
+                  {this.getPatientName()} will be immediately moved into the selected monitoree&apos;s household.
+                </p>
+                <InputGroup size="md">
+                  <InputGroup.Prepend>
+                    <OverlayTrigger overlay={<Tooltip>Search by name or state/local ID.</Tooltip>}>
+                      <InputGroup.Text className="rounded-0 p-1">
+                        <i className="fas fa-search"></i>
+                        <span className="ml-1">Search</span>
+                      </InputGroup.Text>
+                    </OverlayTrigger>
+                  </InputGroup.Prepend>
+                  <Form.Control
+                    id="search-input"
+                    autoComplete="off"
+                    size="md"
+                    name="search"
+                    value={this.state.query.search}
+                    onChange={this.handleSearchChange}
+                    onKeyPress={this.handleKeyPress}
+                  />
+                </InputGroup>
               </Form.Group>
             </Row>
           </Form>
+          <CustomTable
+            columnData={this.state.table.colData}
+            rowData={this.state.table.rowData}
+            totalRows={this.state.table.totalRows}
+            handleTableUpdate={query => this.updateTable({ ...this.state.query, order: query.orderBy, page: query.page, direction: query.sortDirection })}
+            handleEntriesChange={this.handleEntriesChange}
+            isSelectable={false}
+            isEditable={false}
+            isLoading={this.state.isLoading}
+            page={this.state.query.page}
+            handlePageUpdate={this.handlePageUpdate}
+            entryOptions={this.state.entryOptions}
+            entries={this.state.query.entries}
+          />
         </Modal.Body>
         <Modal.Footer>
-          <Button variant="secondary btn-square" onClick={toggle}>
+          <Button id="move-to-household-cancel-button" variant="secondary btn-square" onClick={toggle}>
```

Comment:
this button doesn't reset the state of the modal when you cancel, hit esc, etc.  is that the intended functionality?

Reply:
It should reset - good catch. I've added that functionality in here: fe7f862